web/src/js/ducks/ui/flow.js

```diff
@@ -87,9 +88,31 @@ export function updateEdit(update) {
     return { type: UPDATE_EDIT, update }
 }
 
-export function stopEdit(flow) {
+export function uploadContent(flow, content, type){
     return (dispatch) => {
-        dispatch(flowsActions.update(flow, flow)).then(() => {
+            dispatch(flowsActions.updateContent(flow, content, type)).then( () => {
```

Comment:
We probably just want to listen to the update flow action in the reducer to disable editing mode once the flow is updated by something?